test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -118,10 +120,65 @@ void UcxContext::UcxDisconnectCallback::operator()(ucs_status_t status)
     delete this;
 }
 
-UcxContext::UcxContext(size_t iomsg_size, double connect_timeout, bool use_am) :
-    _context(NULL), _worker(NULL), _listener(NULL), _iomsg_recv_request(NULL),
-    _iomsg_buffer(iomsg_size, '\0'), _connect_timeout(connect_timeout),
-    _use_am(use_am)
+UcxContext::UcpWorkEpoll::UcpWorkEpoll() : _epoll_fd(0), _epoll_event{0}
+{
+    _epoll_fd = epoll_create1(EPOLL_CLOEXEC);
+    assert(_epoll_fd > 0);
+}
+
+UcxContext::UcpWorkEpoll::~UcpWorkEpoll()
+{
+    if (_epoll_fd > 0) {
+        close(_epoll_fd);
+        _epoll_fd = 0;
+    }
+}
+
+void UcxContext::UcpWorkEpoll::add_event(int mon_fd, uint32_t event)
```

Comment:
If we only has fixed event EPOLLIN, then we don't need the parameter uint32_t event.

Reply:
Maybe we should rename add_event since we are not adding event, we are adding some entry to epoll fd list.